downstream/modules/aap-hardening/proc-install-user-pki.adoc

```diff
@@ -13,29 +13,39 @@ Use the following inventory variables to configure the infrastructure components
 
 .PKI certificate inventory variables
 |===
-| *Variable* | *Details*
-| `custom_ca_cert` | The file name of the CA certificate located in the installer directory.
+| *RPM Variable* | *Containerized Variable* | *Details*
+| `custom_ca_cert` | | The file name of the CA certificate located in the installer directory.
 
-| `web_server_ssl_cert` | The file name of the {ControllerName} PKI certificate located in the installer directory.
+Defines a custom CA certificate here when you have the leaf certificates created for each product and need the certificate trust to be established.
 
-| `web_server_ssl_key` | The file name of the {ControllerName} PKI key located in the installer directory.
+| `web_server_ssl_cert` | `controller_tls_cert` | The file name of the {ControllerName} PKI certificate located in the installer directory.
 
-| `automationhub_ssl_cert` | The file name of the {PrivateHubName} PKI certificate located in the installer directory.
+| `web_server_ssl_key` | `controller_tls_key` | The file name of the {ControllerName} PKI key located in the installer directory.
 
-| `automationhub_ssl_key` | The file name of the {PrivateHubName} PKI key located in the installer directory.
+| `automationhub_ssl_cert` | `hub_tls_cert` | The file name of the {PrivateHubName} PKI certificate located in the installer directory.
 
-| `postgres_ssl_cert` | The file name of the database server PKI certificate located in the installer directory. This variable is only needed for the installer-managed database server, not if a third-party database is used.
+| `automationhub_ssl_key` | `hub_tls_key` | The file name of the {PrivateHubName} PKI key located in the installer directory.
 
-| `postgres_ssl_key` | The file name of the database server PKI certificate located in the installer directory. This variable is only needed for the installer-managed database server, not if a third-party database is used.
+| `postgres_ssl_cert` | `postgresql_tls_cert` | The file name of the database server PKI certificate located in the installer directory. This variable is only needed for the installer-managed database server, not if a third-party database is used.
 
-| `automationedacontroller_ssl_cert` | The file name of the {EDAcontroller} PKI certificate located in the installer directory.
+| `postgres_ssl_key` | `postgresql_tls_key` | The file name of the database server PKI certificate located in the installer directory. This variable is only needed for the installer-managed database server, not if a third-party database is used.
 
-| `automationedacontroller_ssl_key` | The file name of the {EDAcontroller} PKI key located in the installer directory.
+| `automationedacontroller_ssl_cert` | `eda_tls_cert` | The file name of the {EDAcontroller} PKI certificate located in the installer directory.
+
+| `automationedacontroller_ssl_key` | `eda_tls_key` | The file name of the {EDAcontroller} PKI key located in the installer directory.
 |===
```

Comment:
entries for `gateway_tls_cert` and `gateway_tls_key` need to be added.  these are the containerized variable names, i'm not sure what the corresponding RPM variable names are.